src/Interpreters/executeQuery.cpp

```diff
@@ -171,6 +178,22 @@ static void logQuery(const String & query, const Context & context, bool interna
                 "OpenTelemetry traceparent '{}'",
                 client_info.client_trace_context.composeTraceparentHeader());
         }
+
+        QueryLogElement elem;
+
+        elem.type = QueryLogElementType::QUERY_START;
+        elem.event_time = current_time_us / 1000000;
+        elem.event_time_microseconds = current_time_us;
+        elem.query_start_time = current_time_us / 1000000;
+        elem.query_start_time_microseconds = current_time_us;
+
+        elem.current_database = context.getCurrentDatabase();
+        elem.query = query;
+        elem.normalized_query_hash = normalizedQueryHash(query);
+
+        elem.client_info = client_info;
+        if (auto query_log = context.getQueryLog())
+            query_log->add(elem);
     }
 }
```

Comment:
There are no tests.

Reply:
Where should be the tests added?

Reply:
Let's fix build first (see the logs for "Fast test"), then add tests.

Reply:
Have I fixed the build test yet?

Reply:
You can check the details by `runlog.out.log` link.

Reply:
The error is thrown by `DB::InterpreterDropQuery::executeToTable`, it seems that it shouldn't caused by my PR.

Reply:
We have no errors in master.

Reply:
Can you help me find out why `DB::Exception: Cannot attach table with UUID 46c4cb82-ae3e-4b5b-bcfe-28cc72677394, because it was detached but still used by some query` will be thrown?